Changes.md

```diff
@@ -21,6 +21,7 @@ Improvements
   - Improved formatting of column headers containing whitespace.
   - The "Double-click to toggle" tooltip is no longer displayed while hovering over non-editable cells, and a "Double-click to edit" tooltip is now displayed while hovering over other non-toggleable but editable cells.
 - Spreadsheet : Added yellow underlining to the currently active row.
+- usdview : Added Windows support.
```

Comment:
Is this #5599? If so, can we bill it is a fix rather than improvement and include a reference to the issue number? We could also put "Fixes #5599" in the commit message...

Reply:
Yes, that fixes #5599. Fixed in https://github.com/GafferHQ/gaffer/pull/6044/commits/c145aa6f04f706267ffda844f3f62302b216d865